src/angularfire2.spec.ts

```diff
@@ -15,80 +15,83 @@ import {
   AngularFire,
   FirebaseObjectObservable,
   FIREBASE_PROVIDERS,
-  FirebaseAuth,
-  FirebaseUrl,
-  FirebaseRef,
+  AngularFireAuth,
+  FirebaseConfig,
+  FirebaseApp,
   defaultFirebase,
-  FirebaseDatabase
+  FirebaseDatabase,
+  FirebaseAppConfig
 } from './angularfire2';
```

Comment:
Add a todo comment to remove FirebaseAppConfig once it's included in the standard typings.